src/main/java/Study_hw_1/CarManager.java

```diff
@@ -0,0 +1,63 @@
+package Study_hw_1;
+
+import java.util.Scanner;
+
+public class CarManager {
+    private Car user[];
```

Comment:
C 스타일의 배열 선언이네요. 굳이 배열로 쓸 거라면 Car[] user; 가 좋겠네요.

자바는 객체지향 언어입니다. 데이터를 배열로 다루기보다, 컬렉션으로 관리하면 어떨까요?
```suggestion
    private List<Car> users = new ArrayList<>();
```